lesson07/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+    <script src="./index.js" defer></script>
+  </head>
+  <body>
+    <div>
+      <div id="js-loading"></div>
```

Comment:
Try to use only `ul` for html.

Reply:
Remove div and move loading into ul.
[754f3c5](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/7/commits/754f3c56d7bc346738cec6500a9f3c26e84e7614)